helpdesk-service/helpdesk-service-server/src/test/resources/jbehave.stories/makerCheckerEndpoints.story

```diff
@@ -0,0 +1,48 @@
+Meta:
+
+Narrative:
+As a user
+I want to create update and read a makerchecker
+So that I can achieve a business goal
```

Comment:
As a user
I want to save and recover maker/checker data
So that I can assert those actions are available and working.

Reply:
ok